### PR TITLE
add mongodb to stack for prototyping

### PR DESCRIPTION
**Pull request by vpchung**
What was done:
- `db` parameter was added to docker-compose.yml, using the `nlpsandbox/mongo:0.1.0` image
- environment variables were added in .env file
- environment variables are called in docker-compose.yml

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MongoDB
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=admin
+MONGO_INITDB_ROOT_PASSWORD=password
+
+## Hardened MongoDB
+MONGO_INITDB_DATABASE=nlpsandbox
```
Comment (thomasyu888) [inline .env:7]: should these be changed from `nlp` to `rocc`?

Comment (tschaffter) [reply .env:7]: Yes

Comment (tschaffter) [reply .env:7]: Use `MONGO_INITDB_DATABASE=rocc`

File: .env [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+MONGO_USERNAME=nlpmongo
```
Comment (tschaffter) [inline .env:8]: Replace MONGO_USERNAME AND MONGO_PASSWORD by `roccmongo`

File: .env [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# API SERVER
+SERVER_PROTOCOL=http://
+SERVER_DOMAIN=localhost
+SERVER_PORT=8080
+
+## DB
+DB_PROTOCOL=mongodb://
+DB_DOMAIN=localhost
+DB_PORT=27017
+DB_DATABASE=nlpsandbox
```
Comment (tschaffter) [inline .env:20]: DB_DATABASE, DB_USERNAME and DB_PASSWORD should take the same value as MONGO_INITDB_DATABASE, MONGO_USERNAME and MONGO_PASSWORD respectively.

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (vpchung) [reply .env:7]: Ah okay, wasn't sure if I could change these values.

**Review by tschaffter — APPROVED** [review]
Looks good!

**Pull request merged**